src/api/java/jni/solver.cpp

```diff
@@ -2121,29 +2141,70 @@ JNIEXPORT jlong JNICALL Java_io_github_cvc5_Solver_getValueSepNil(
  * Method:    declarePool
  * Signature: (Ljava/lang/String;J[J])J
  */
-JNIEXPORT jlong JNICALL Java_io_github_cvc5_Solver_declarePool(
-    JNIEnv* env, jobject, jlong pointer, jstring jsymbol, jlong sort, jlongArray initValuePointers)
+JNIEXPORT jlong JNICALL
+Java_io_github_cvc5_Solver_declarePool(JNIEnv* env,
+                                       jobject,
+                                       jlong pointer,
+                                       jstring jsymbol,
+                                       jlong sort,
+                                       jlongArray initValuePointers)
 {
   CVC5_JAVA_API_TRY_CATCH_BEGIN;
   Solver* solver = reinterpret_cast<Solver*>(pointer);
   const char* s = env->GetStringUTFChars(jsymbol, nullptr);
   std::string symbol(s);
   Sort* sortptr = reinterpret_cast<Sort*>(sort);
-  std::vector<Term> initValue = getObjectsFromPointers<Term>(env, initValuePointers);
+  std::vector<Term> initValue =
+      getObjectsFromPointers<Term>(env, initValuePointers);
```

Comment:
To make future PRs easier to review, please don't change the formatting of unrelated code in the same PR.

Reply:
Unlike other cvc5 files, I made sure this file was clang-formatted from the start. 
Somehow, its format became broken with new commits. 
This means the flow used to format these files  is not working properly. 
The same applies for all java files, which I don't think are applied to new commits.